### PR TITLE
Add script for removing ignored key from config.json in tracks

### PR DESCRIPTION
**Pull request by kytrinyx**
This can be run for https://github.com/exercism/meta/issues/3 once we've released the new version of configlet.

### CODE REVIEW

File: scripts/ad-hoc/rm-ignored/run [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  system("git checkout master")
+  system("git pull --rebase origin master")
+  system("git checkout -b %s" % branch_name)
+
+  config = JSON.parse(File.read("config.json"))
+  config.delete("ignored")
+  File.open("config.json", "w") do |f|
+    f.puts JSON.pretty_generate(config)
+  end
```
Comment (Insti) [inline scripts/ad-hoc/rm-ignored/run]: Are there any `config.json`s that don't contain an `ignored` key?

File: scripts/ad-hoc/rm-ignored/run [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  system "git add config.json"
+
+  system "git commit -m \"%s\"" % commit_message
+  system "git push origin %s" % branch_name
+  system "hub pull-request -m \"%s\"" % pull_request_message
```
Comment (Insti) [inline scripts/ad-hoc/rm-ignored/run]: Maybe join all these with ' && ' and just call `system` once?
This has the potential to submit broken PRs.

Comment (kytrinyx) [reply scripts/ad-hoc/rm-ignored/run]: 👍

File: scripts/ad-hoc/rm-ignored/run (hunk 1 above)
Comment (kytrinyx) [reply scripts/ad-hoc/rm-ignored/run]: I'll check, but I don't think so.

Comment (kytrinyx) [reply scripts/ad-hoc/rm-ignored/run]: Actually, I think I'll just put a guard around it. Doesn't hurt.

**Comment by kytrinyx** [comment]
I rejiggered the script a bit. Moved the actual work into a method, and then I return a string if there's a problem, and print all the potential problems at the end (the script is noisy, I'm not hiding any of the regular command output to stdout, so if I just print the problems as I go, then I probably won't see them when I run it).

**Comment by kytrinyx** [comment]
I've released a new configlet (v2.0.0)

**Comment by kytrinyx** [comment]
Thanks for the help, @Insti, I'm going to go ahead and merge it and run it.

**Pull request merged**